jordan_bundy/server.js

```diff
@@ -0,0 +1,24 @@
+var http = require('http');
+var app = require('express')();
+var fs = require('fs');
+
+app.get('/data', function(req, res) {
+  var readStream = fs.createReadStream(__dirname + '/data/input.json');
+  res.writeHead(200, {'content-type':'application/json'});
+  readStream.pipe(res);
+});
+
+app.post('/data', function(req, res) {
+  var writeStream = fs.createWriteStream(__dirname + '/data/input.json');
+
```

Comment:
Thanks Molly, I must have misunderstood Tyler.  I was working on that, and heard him say don't worry about updating the file.  I like the idea of being able to update the resource though, so I'll go back and complete this after this week.